Podfile.lock

```diff
@@ -5,121 +5,143 @@ PODS:
+    - GoogleUtilities/AppDelegateSwizzler (~> 7.4)
+    - GoogleUtilities/MethodSwizzler (~> 7.4)
+    - GoogleUtilities/Network (~> 7.4)
+    - "GoogleUtilities/NSData+zlib (~> 7.4)"
+    - nanopb (~> 2.30908.0)
+  - GoogleAppMeasurement/AdIdSupport (8.7.0):
+    - GoogleUtilities/AppDelegateSwizzler (~> 7.4)
+    - GoogleUtilities/MethodSwizzler (~> 7.4)
+    - GoogleUtilities/Network (~> 7.4)
+    - "GoogleUtilities/NSData+zlib (~> 7.4)"
+    - nanopb (~> 2.30908.0)
+  - GoogleDataTransport (9.1.0):
+    - GoogleUtilities/Environment (~> 7.2)
+    - nanopb (~> 2.30908.0)
+    - PromisesObjC (< 3.0, >= 1.2)
+  - GoogleSignIn (6.0.2):
+    - AppAuth (~> 1.4)
     - GTMAppAuth (~> 1.0)
     - GTMSessionFetcher/Core (~> 1.1)
-  - GoogleUtilities/AppDelegateSwizzler (6.7.2):
+  - GoogleUtilities/AppDelegateSwizzler (7.5.2):
     - GoogleUtilities/Environment
     - GoogleUtilities/Logger
     - GoogleUtilities/Network
-  - GoogleUtilities/Environment (6.7.2):
-    - PromisesObjC (~> 1.2)
-  - GoogleUtilities/Logger (6.7.2):
+  - GoogleUtilities/Environment (7.5.2):
+    - PromisesObjC (< 3.0, >= 1.2)
+  - GoogleUtilities/Logger (7.5.2):
     - GoogleUtilities/Environment
-  - GoogleUtilities/MethodSwizzler (6.7.2):
+  - GoogleUtilities/MethodSwizzler (7.5.2):
     - GoogleUtilities/Logger
-  - GoogleUtilities/Network (6.7.2):
+  - GoogleUtilities/Network (7.5.2):
     - GoogleUtilities/Logger
     - "GoogleUtilities/NSData+zlib"
     - GoogleUtilities/Reachability
-  - "GoogleUtilities/NSData+zlib (6.7.2)"
-  - GoogleUtilities/Reachability (6.7.2):
+  - "GoogleUtilities/NSData+zlib (7.5.2)"
+  - GoogleUtilities/Reachability (7.5.2):
     - GoogleUtilities/Logger
-  - GoogleUtilities/UserDefaults (6.7.2):
+  - GoogleUtilities/UserDefaults (7.5.2):
     - GoogleUtilities/Logger
   - GTMAppAuth (1.2.2):
     - AppAuth/Core (~> 1.4)
     - GTMSessionFetcher/Core (~> 1.5)
-  - GTMSessionFetcher/Core (1.5.0)
+  - GTMSessionFetcher/Core (1.7.0)
   - IGListDiffKit (4.0.0)
   - KeychainAccess (4.2.2)
-  - nanopb (1.30906.0):
-    - nanopb/decode (= 1.30906.0)
-    - nanopb/encode (= 1.30906.0)
-  - nanopb/decode (1.30906.0)
-  - nanopb/encode (1.30906.0)
+  - nanopb (2.30908.0):
+    - nanopb/decode (= 2.30908.0)
+    - nanopb/encode (= 2.30908.0)
+  - nanopb/decode (2.30908.0)
+  - nanopb/encode (2.30908.0)
   - Nimble (9.2.0)
-  - PromisesObjC (1.2.12)
+  - PromisesObjC (2.0.0)
   - Quick (3.1.2)
-  - TwilioVideo (4.5.0)
+  - TwilioVideo (4.6.0)
 
 DEPENDENCIES:
   - Alamofire (~> 5)
-  - AppCenter/Distribute (~> 3)
-  - Firebase/Analytics (~> 6)
-  - Firebase/Crashlytics (~> 6)
-  - FirebaseUI/Auth (~> 9)
-  - FirebaseUI/Google (~> 9)
+  - AppCenter/Distribute (~> 4)
+  - Firebase/Analytics (~> 8)
+  - Firebase/Crashlytics (~> 8)
+  - FirebaseUI/Auth (~> 12)
+  - FirebaseUI/Google (~> 12)
   - IGListDiffKit (~> 4)
   - KeychainAccess (~> 4)
   - Nimble (~> 9)
   - Quick (~> 3)
-  - TwilioVideo (~> 4.5)
+  - TwilioVideo (~> 4)
```

Comment:
4.6?

Reply:
I think 4 is best. This is what we do for all other dependencies (pick up all non-breaking changes) and I'm not aware of a reason why the video SDK would be any different. 

It was changed to 4.5 in a recent PR but it should really just be 4 I think.

Reply:
> It was changed to 4.5 in a recent PR but it should really just be 4 I think.

Different schools of thought. I would prefer ~> 4.5 since the app is incompatible with earlier versions (it calls APIs introduced in 4.5). But the Podfile.lock is committed and is enforcing the version requirement. So, I don't mind leaving it as is. 👍

Reply:
Oh I see what you're saying. That is interesting and should think about it. But for now I tend to think just use major version like other dependencies.